product_docs/docs/epas/15/deployment_options/index.mdx

```diff
@@ -2,12 +2,12 @@
 title: "Deployment options"
 ---
 
-You can deploy and install EDB products using the following methods:
+You can deploy and install EDB Postgres Advanced Server using:
 
-- [BigAnimal](/biganimal/latest) is a fully managed database-as-a-service with built-in Oracle compatibility, running in your cloud account and operated by the Postgres experts. BigAnimal makes it easy to set up, manage, and scale your databases. Provision PostgreSQL or EDB Postgres Advanced Server with Oracle compatibility.
+- [BigAnimal](/biganimal/latest), a fully managed database-as-a-service with built-in Oracle compatibility. It runs in your cloud account and is operated by the Postgres experts. BigAnimal makes it easy to set up, manage, and scale your databases. Provision PostgreSQL or EDB Postgres Advanced Server with Oracle compatibility.
 
-- [EDB PostgreSQL for Kubernetes](/postgres_for_kubernetes/latest/) is an operator designed by EnterpriseDB to manage PostgreSQL workloads on any supported Kubernetes cluster running in private, public, hybrid, or multi-cloud environments. EDB PostgreSQL for Kubernetes adheres to DevOps principles and concepts such as declarative configuration and immutable infrastructure.
+- [EDB PostgreSQL for Kubernetes](/postgres_for_kubernetes/latest/), an operator designed by EnterpriseDB to manage PostgreSQL workloads on any supported Kubernetes cluster running in private, public, hybrid, or multi-cloud environments. EDB PostgreSQL for Kubernetes adheres to DevOps principles and concepts such as declarative configuration and immutable infrastructure.
 
-- Installation using native packages or installers:
+- [EDB Postgres Advanced Server AMI](aws_epas), which is an Amazon Machine Image containing EDB Postgres Advanced Server. It's available from Amazon Marketplace.
 
-   - [Installing EDB Postgres Advanced Server](/epas/latest/installing)
\ No newline at end of file
+- Native packages or installers. See [Installing EDB Postgres Advanced Server](/epas/latest/installing).
```

Comment:
I removed the subbullet because there weren't any other items in the list, so it made more sense to run this in.

Reply:
I am still seeing this bullet in latest deployment